Philips.CodeAnalysis.MaintainabilityAnalyzers/Maintainability/OperatorsCountAnalyzer.cs

```diff
@@ -0,0 +1,93 @@
+﻿// © 2023 Koninklijke Philips N.V. See License.md in the project root for license information.
+
+using Microsoft.CodeAnalysis;
+using Microsoft.CodeAnalysis.CSharp;
+using Microsoft.CodeAnalysis.Diagnostics;
+using System.Collections.Immutable;
+
+using Microsoft.CodeAnalysis.CSharp.Syntax;
+
+using Philips.CodeAnalysis.Common;
+
+namespace Philips.CodeAnalysis.MaintainabilityAnalyzers.Maintainability
+{
+	/// <summary>
+	/// Diagnostic for inconsistent number of operators in a class.
+	/// </summary>
+	[DiagnosticAnalyzer(LanguageNames.CSharp)]
+	public class OperatorsCountAnalyzer : DiagnosticAnalyzer
+	{
+		private const string PlusMinusTitle = "Align number of + and - operators.";
+		private const string PlusMinusMessage = "Align number of + and - operators.";
+		private const string PlusMinusDescription = 
+			"A class should have the same number of + as - operators and these shall have the same arguments, in the same order.";
+		private const string MultiplyDivideTitle = "Align number of * and / operators.";
+		private const string MultiplyDivideMessage = "Align number of * and / operators.";
+		private const string MultiplyDivideDescription =
+			"A class should have the same number of * as / operators and these shall have the same arguments, in the same order.";
```

Comment:
We're not enforcing same arguments/same order. Idk if we want to put it in the description or not.

Reply:
I'll drop it, as I dropped it also in the implementation (for runtime performance reasons)